episodes/06-blurring.md

```diff
@@ -327,7 +327,143 @@ fig, ax = plt.subplots()
 plt.imshow(blurred)
 ```
 
-![](fig/gaussian-blurred.png){alt='Original image'}
+![](fig/gaussian-blurred.png){alt='Blurred image'}
+
+
+## Visualizing Blurring
+
+Somebody said once "an image is worth a thousand words". 
+What is actually happening to the image pixels when we apply blurring may be 
+difficult to grasp, let's now visualize the effects of blurring from a different
+perspective.
+
+Let's use the petri-dish image from previous episodes:
+
+![
+Graysacle version of the Petri dish image
+](fig/petri-dish.png){alt='Bacteria colony'}
+
+What we want to see here is the pixel intensities from a lateral perspective,
+we want to see the profile of intensities .
+For instance, let's look for the intensities of the pixels along the horizontal
+directory at `Y=150`:
```

Comment:
I am not sure about the use of "directory" here: I don't think this has been used to describe an axis or dimension of an image elsewhere in the lesson.

Reply:
Definitely a typo, I guess "direction" should have gone there instead... I think "line" would work (better than "direction")